docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1,182 @@
-Ubuntu specific instructions
\ No newline at end of file
+# Installing LORIS In-Depth
+
+This guide assumes an installation on an Ubuntu server. 
+
+If you are using CentOS, please visit the CentOS installation guide for 
+instructions on installation.
+
+We also have instructions for how to set up LORIS for macOS. However, only
+ Ubuntu and CentOS are officially supported by the team so we may be unable to
+ help you with issues arising on other systems.
+
+
+When you've completed this guide, further set up is required. Please visit
+[the Setup page](https://github.com/aces/Loris/wiki/Setup) in order to be
+able to use LORIS properly.
+
+## Prequisities
+
+### Server Stack
+
+LORIS requires a LAMP stack in order to run, specifically:
+* Apache 2.4
+* MySQL 5.7 (or MariaDB 10.1) (or higher)
+* PHP 7.3 (or higher)
+
+Additionally, LORIS requires the following package managers:
+* npm 8.0 (or higher)
+* composer
+
+This guide does not cover installation of these requirements.
+
+### Apt Packages
+The following Ubuntu packages are required and should be installed using 
+`sudo apt install ...`.
+
+* git
+* curl
+* wget
+* zip
+* unzip
+* php-json
+* make
+* software-properties-common
+* php7.3-mysql
+* php7.3-xml
+* php7.3-json
+* php7.3-mbstring
+* php7.3-gd
+* libapache2-mod-php7.3
+
+### Database set up
+
+MySQL (or MariaDB) must be installed and a root or admin-level MySQL user must
```

Comment:
Because it's tremendously useful for support and troubleshooting, I think we should keep using the placeholder names for these accounts/roles: `lorisadmin` and `lorisdbadmin`.  
Is there a reason they're not here? 

By all means let's improve upon them if we can (e.g. maybe `dbadmin` because `lorisdbadmin` is too close to the unix user `lorisadmin`?)

Reply:
Not really comfortable resolving this conversation, since i'd like these comments to be retained on the PR...